third_party/cares/cares.BUILD

```diff
@@ -159,9 +159,15 @@ cc_library(
         "inet_net_pton.c",
         "inet_ntop.c",
         "windows_port.c",
-    ],
+    ] + select({
```

Comment:
Why is there a need to selectively include this? The header guards should be taking care of not compiling these sources outside Android?